### DISCUSSION TITLE
Clock 2.6

### DISCUSSION THREAD

**Discussion by BlackyHawky** [BlackyHawky/Clock#70] [Announcements]
**Fixes**
- Fix "Gradually increase volume" summary for timer;
- Fix digital clock display in screensaver (this mainly concerns devices where the display and font size is set to large in the device settings);

**Improvements**
- Remove unnecessary permissions;
- Set alarm notification reminder time;
- Dismiss alarms at the end of the ringtone and add alarm snooze actions;
- Add setting to enable vibrations when creating alarms;
- Update hints when alarm buttons are clicked in expired alarms animation;
- Don't create alarms automatically after app installation;
- Add setting to keep the screen on if a timer is displayed;
- Set default time to add to timer in the timer settings;
- Set a custom time to add to the timer by long-pressing the add button;
- Improve alarm and timer views:
  - Remove ripple effect of the bedtime alarm title;
  - To be consistent, alarm and timer titles are the same size;
  - Adjust the padding for timers to be identical to the padding for alarms;
  - For Tablets, the Reset timer button can no longer be hidden by the floating action button;
  - For tablets, Play/Pause and Add time buttons are no longer minimized, as is the case with phones in landscape mode;
  - For phones in landscape mode, the size of the remaining time has been slightly reduced and timer sizes are slightly longer;
  - Add a ripple effect to the Reset button when it is pressed;
- Display the preferences summary in its entirety (this mainly concerns devices where the display and font size is set to large in the device settings);
- Recreate the main activity only if specific settings are changed;

---

**Note:**

**UNFORTUNATELY, DUE TO A CHANGE IN THE DATABASE, YOU'LL NEED TO REINSTALL THE APP**

<hr /><em>This discussion was created from the release <a href='https://github.com/BlackyHawky/Clock/releases/tag/2.6'>Clock 2.6</a>.</em>

**Comment by qw123wh** [comment]
I noticed that by pausing the timers, they don't become "smaller" as before, practically now with version 2.6 they are always "expanded". This behavior should have only been expected in tablets right?

**Reply by BlackyHawky** [reply]
Timers are only reduced if stopped.
They have never been reduced when on pause.
Indeed, this only applies to phones, not tablets.

**Reply by qw123wh** [reply]
Yes, sorry my mistake. The behavior is right.

**Comment by sunglocto** [comment]
welp, there goes my 6000 hour stopwatch

**Reply by BlackyHawky** [reply]
I don't understand what that means... But why not 😅
